crates/telnet-host/src/main.rs

```diff
@@ -47,6 +47,9 @@ struct Args {
         default_value = "tcp://0.0.0.0:7898"
     )]
     narrative_server: String,
+
+    #[arg(long, help = "Enable debug logging", default_value = "false")]
```

Comment:
Didn't this change already get made in another commit?  Need a rebase?

Reply:
Negative, that was `daemon`, this is `telnet-host`.